distribution/lib/Standard/Database/0.0.0-dev/src/Data/Column.enso

```diff
@@ -317,6 +329,22 @@ type Column
     < : Column | Any -> Column
     < self other = self.make_binary_op "<" other new_type=SQL_Type.boolean
 
+    ## Element-wise inclusive bounds check.
+
+       Arguments:
+       - lower: The lower bound to compare elements of `self` against. If
+         `lower` is a column, the comparison is performed pairwise between
+         corresponding elements of `self` and `lower`.
+       - upper: The upper bound to compare elements of `self` against. If
+         `upper` is a column, the comparison is performed pairwise between
+         corresponding elements of `self` and `upper`.
+
+       Returns a column with boolean values indicating whether values of this
+       column fit between the lower and upper bounds (both ends inclusive).
+    between : (Column | Any) -> (Column | Any) -> Column
+    between self lower upper =
```

Comment:
suggestion: maybe we could think of adding a parameter if it shouldn't be inclusive by default

Reply:
The [SQL BETWEEN](https://docs.oracle.com/cd/E11882_01/server.112/e41084/conditions011.htm#SQLRF52164) this is more or less supposed to emulate is always inclusive. So if we added an option, we'd have to make it unsupported on Database backend or emulate it using regular comparison operators. 

But we already can always just apply two filters: Greater+Less to get an exclusive between.

So I'm not sure if it is necessary to add it, I guess it's mostly a question to @jdunkerley who designed the original set of `Filter_Condition`s.

Reply:
was aiming to keep BETWEEN simple and just match SQLs behavior with a view that is what our users would expect.